java-checks-test-sources/src/main/files/non-compiling/checks/aws/AwsRegionShouldBeSetExplicitlyCheck.java

```diff
@@ -0,0 +1,8 @@
+package checks.aws;
```

Comment:
Correct me if I'm wrong - if we introduce this file, don't we also need a corresponding test that uses `.onFiles(nonCompilingTestSourcesPath("checks/aws/AwsRegionShouldBeSetExplicitlyCheck.java"))`?